libs/langchain/langchain/chains/conversational_retrieval/base.py

```diff
@@ -135,8 +135,8 @@ def _call(
         else:
             docs = self._get_docs(new_question, inputs)  # type: ignore[call-arg]
```

Comment:
this technically changes chain behavior, as the new question was being used for document retrieval whether or not it was passed into final chain for answering. i'm actually not sure what the intended behavior here is, cc @hwchase17

Reply:
if we do want to use the new question for rertieval in all cases then i do think `rephrase_question` is a confusing parameter name. maybe something like `pass_rephrased_question_to_llm` instead

Reply:
yeah this is just a different chain. this should maybe be a separate chain entirely rather than messing with the internals. this is not a bug, it is intentional